scripts/codebuild

```diff
@@ -6,30 +6,67 @@ set -x
 echo "${DOCKER_PASSWORD}" | \
   docker login --username "${DOCKER_USERNAME}" --password-stdin
 
-#LOCUSTFILE can be set in the build overrides of the CodeBuild config
-docker build -t "${ECR_REPO}:exp" --build-arg locustfile="${LOCUSTFILE}" .
+tag=${CODEBUILD_RESOLVED_SOURCE_VERSION:-exp}
```

Comment:
The Codebuild docs say this would be the git commit hash I think but I can't match up the value to a commit in this PR.

<img width="1232" alt="Screen Shot 2021-11-22 at 10 39 30 AM" src="https://user-images.githubusercontent.com/52669884/142890681-1e0d9250-e0c5-4991-9c81-d8e796fce5ce.png">

https://docs.aws.amazon.com/codebuild/latest/userguide/build-env-ref-env-vars.html

Reply:
Whoops I see it now GitHub just had difficulty searching for it maybe because of the force push https://github.com/transcom/milmove_load_testing/commit/f6af768fea72d92ea6e95b24ab9b057d1c44213e

Reply:
Sorry for confusing things. I guess I should add a comment explaining `CODEBUILD_RESOLVED_SOURCE_VERSION`

Reply:
I just added a comment so hope that helps